README.md

```diff
@@ -40,6 +40,7 @@ You need the following permissions to run this module.
 - [ Complete example with byok encryption, CBR rules and storing credentials in secrets manager](examples/complete)
 - [ Default example](examples/default)
 - [ Financial Services Cloud profile example](examples/fscloud)
+- [ Restored DB example](examples/restored)
```

Comment:
I'd name this "Restore from backup example" to match the name of the actual functionality

Also the name of the folder would be "backup" instead of "restored"